source/jobs/prerequisites/oracle.txt

```diff
@@ -33,7 +33,14 @@ About this Task
   architecture, see `Overview of Container Databases and Pluggable Databases <https://oracle-base.com/articles/12c/multitenant-overview-container-database-cdb-12cr1>`__.
 - Some commands differ based on whether the database is single or
   multi-tenant. In a multi-tenant database, permissions must
-  include the suffix ``CONTAINER=ALL``.
+  include the suffix ``CONTAINER=ALL``. To examine whether the database is 
+  multi-tenant or not, run the following SQL query:
```

Comment:
personal take on the previous feedback (but take it with a pinch of salt): I would not phrase it like this

> To examine whether the database is multi-tenant or not

for a simple check / confirmation. this is not an exploration or a detailed investigation but rather a very simple check "true / false"

i think _"to check"_ would be a better fit here